contracts/clients/src/BlsSigner.ts

```diff
@@ -200,17 +200,56 @@ export default class BlsSigner extends Signer {
     throw new Error("_signTypedData() is not implemented.");
   }
 
-  connectUnchecked(): ethers.providers.JsonRpcSigner {
-    throw new Error("connectUnchecked() is not implemented.");
+  connectUnchecked(): BlsSigner {
+    throw new Error(
+      "connectUnchecked() is not implemented. Use connectBlsUnchecked instead.",
+    );
+  }
+
+  // When we use regular "connectUnchecked", we need to pass the private key into the method as that is a required
+  // argument for the BlsSigner constructor. This breaks liskov substitution as the subclass method signature is different.
```

Comment:
In a previous PR, there was an agreement that adding custom methods to be used in place of an ethers method was undesirable. So I've created this method apprehensively, but I'm unsure how we could get `connectUnchecked` to work due to needing a different method signature. Open to suggestions on better ways to resolve this.

Reply:
https://github.com/web3well/bls-wallet/pull/422/files#r1044853129